test/llvm/PREDIFF

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+TESTNAME=$1
+OUTFILE=$2
+
+TMPFILE="$outfile.prediff.tmp"
+
+FILECHECK=${CHPL_HOME}/third-party/llvm/install/linux64-gnu/bin/FileCheck
+
+mv $OUTFILE $TMPFILE
+$FILECHECK --input-file $TMPFILE $TESTNAME.chpl 2> $OUTFILE
+rm $TMPFILE
```

Comment:
I think this PR would be best merged after I add FileCheck to Chapel's test system and then refactor it, because current way we use FileCheck is actually a fast hack to see whether it makes sense to use this tool (it makes!).